### PR TITLE
Updated the example to remove openshift/fabric8 aspects, so just beco…

### PR DESCRIPTION
**Pull request by objectiser**
…me a simple spring-boot app using OT JavaAgent with Hawkular APM

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkroehling Could you review please?

I've simplified each service by removing the openshift/fabric8 aspects, and downloading the agent so doesn't need to be done manually.

Aim is to make this a simple example of using OT JavaAgent (with spring boot) against a real Tracer (e.g. Hawkular APM) - so can be referenced from OT JavaAgent repo's README.

Currently still dependent upon some snapshots, but hopefully good enough to review against the above aim.

Once using non-snapshot versions, may see about getting it moved to the hawkular org.

File: README.md [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
+[this PR](https://github.com/opentracing-contrib/java-agent/pull/1). 
 
-NOTE: This example is currently dependent upon [this PR](https://github.com/opentracing-contrib/java-web-servlet-filter/pull/11) therefore this version will need to be cloned and built locally.
+NOTE: This example is also currently dependent upon [this PR](https://github.com/opentracing-contrib/java-web-servlet-filter/pull/11) therefore this version will need to be cloned and built locally.
 
 ## The Example
 
-The example services are based on [this example](http://github.com/obsidian-toaster/quick_rest_springboot-tomcat),
-but modified to have two spring boot services, one calling the other.
+The example has two spring boot services. The first service (Service A) is called by an external client (e.g. curl) using the `/greeting` endpoint, supplying an optional `name` parameter. This service then calls Service B to obtain the message template to be used. Finally the template, with the optional supplied `name` parameter is used to return a message to the client.
```
Comment (jpkrohling) [inline README.md]: `s/spring boot/Spring Boot/`

File: README.md [hunk 2]
```diff
@@ -1,44 +1,33 @@
...
-To try out the example, without any instrumentation being added, build each service:
+Before being able to run the example with the OpenTracing Java Agent, it will be necessary to [startup a Hawkular APM server](https://hawkular.gitbooks.io/hawkular-apm-user-guide/content/quickstart/):
 
 ```
-cd servicea
-mvn clean install
-cd ../serviceb
-mvn clean install
-cd ..
+docker run -p 8180:8080 jboss/hawkular-apm-server-dev
```
Comment (jpkrohling) [inline README.md:15]: \o/

File: README.md [hunk 3]
```diff
@@ -1,44 +1,33 @@
...
 ```
 
-then run both services - you will need to start two separate command windows and then run the following
-commands in each:
+For this example, we will run the Hawkular APM server on port 8180, and the example on port 8080.
```
Comment (jpkrohling) [inline README.md]: ... and the examples on ports 8080 and 8090

File: README.md [hunk 4]
```diff
@@ -90,6 +53,45 @@ Selecting the instances button will also show the individual trace instance.
 This section describes how to take a standard Spring Boot application and prepare it for instrumentation
 using the OpenTracing Java Agent.
 
+### Obtain the OpenTracing Java Agent
+
+This can be done manually, and stored in an appropriate location to be accessed when invoking the service. However for simplicity we have included retrieving the agent as part of the example build:
```
Comment (jpkrohling) [inline README.md:58]: Would it be worth adding something like "The code below is incomplete and may be outdated. See the example's `pom.xml` for an actual example."?

**Review by jpkrohling — APPROVED** [review]
LGTM. I added a couple of comments, but nothing serious. This can be merged as is.

File: README.md (hunk 4 above)
Comment (objectiser) [reply README.md:58]: I think I'll leave this as should be ok for now.

**Comment by objectiser** [comment]
@jpkrohling Thanks!

**Pull request merged**